docs/user/gcp/install_upi.md

```diff
@@ -87,6 +93,7 @@ open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 ### Remove DNS Zones (Optional)
 If you don't want [the ingress operator][ingress-operator] to create DNS records on your behalf, remove the `privateZone` and `publicZone` sections from the DNS configuration.
 If you do so, you'll need to [add ingress DNS records manually](#add-the-ingress-dns-records) later on.
+This is recommended if you are installing a cluster into a Shared VPC unless the master/worker service accounts have access to update DNS records in the host project.
```

Comment:
afaik, this is incorrect, the master/worker SAs don't interact with dns zones, but the ingress-operator SAs

Reply:
Updated with 998a518a17e54a792bd52cf31b217775eb20f5a3